File: app/src/main/java/com/mycompany/myapp/monitoring/LoggingOnlyCrashReporter.java
package com.mycompany.myapp.monitoring;

import timber.log.Timber;

public class LoggingOnlyCrashReporter implements CrashReporter {
    @Override
    public void logMessage(String message) {
        Timber.i(message);
    }

    @Override
    public void logException(String message, Exception ex) {
        Timber.e(ex, message);
    }
}

File: app/src/main/java/com/mycompany/myapp/monitoring/LoggingOnlyCrashReporter.kt
package com.mycompany.myapp.monitoring

import timber.log.Timber

class LoggingOnlyCrashReporter : CrashReporter {
    override fun logMessage(message: String) = Timber.i(message)

    override fun logException(message: String, ex: Exception) = Timber.e(ex, message)
}
